csrc/python_frontend/python_bindings.cpp

```diff
@@ -3266,5 +3308,63 @@ void initNvFuserPythonBindings(PyObject* module) {
             !isResharding(tv->definition()));
       },
       py::arg("tensor"));
+  nvf_sched.def(
+      "can_schedule",
+      [](FusionDefinition::SchedOperators& self,
+         const ScheduleHeuristic& heuristic) {
+        NVF_CHECK(
+            self.validUse(),
+            "Attempting to use a SchedOperators Op prior to definition!");
+        return canSchedule(self.fusion_definition->userSchedule(), heuristic);
+      },
+      py::arg("heuristic"));
+  nvf_sched.def(
+      "find_compatible_schedulers", [](FusionDefinition::SchedOperators& self) {
+        NVF_CHECK(
+            self.validUse(),
+            "Attempting to use a SchedOperators Op prior to definition!");
+
+        Fusion* fusion = self.fusion_definition->userSchedule()->schedule.get();
+        NVF_ERROR(
+            fusion != nullptr, "Requires Fusion to use heuristic schedulers");
+
+        SchedulerRuntimeInfo* runtime_info =
+            self.fusion_definition->userSchedule()->runtime_info.get();
+        NVF_ERROR(
+            runtime_info != nullptr,
+            "Requires SchedulerRuntimeInfo to use heuristic schedulers");
```

Comment:
nitpick. we are using similar checks in `canSchedule`.
```
  NVF_ERROR(
      sched->schedule != nullptr,
      "Requires Fusion to use heuristic schedulers");
  NVF_ERROR(
      sched->runtime_info != nullptr,
      "Requires SchedulerRuntimeInfo to use heuristic schedulers");
```

Maybe put this as a method in `UserSchedule`

Reply:
I added separate get functions for `Fusion*` and `SchedulerRuntimeInfo*` that includes these checks respectively.